### PR TITLE
Update leftover review comments from adding property type constraints

### PR DESCRIPTION
**Pull request by Hunterness**
Addresses leftover comments on https://github.com/neo4j/docs-cypher/pull/597

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
Looks like you've updated the documentation!

Check out your changes at https://neo4j-docs-cypher-619.surge.sh

File: modules/ROOT/pages/constraints/examples.adoc [hunk 1]
```diff
@@ -288,8 +288,8 @@ CREATE (book:Book {isbn: '9780763630188', title: 'Moby Dick'})
 CREATE CONSTRAINT book_title FOR (book:Book) REQUIRE book.title IS UNIQUE
 ----
 
-In this case the constraint can not be created because it is violated by existing data.
-Either use xref::indexes-for-search-performance.adoc[] instead, or remove the offending nodes and then re-apply the constraint.
+In this case, the constraint cannot be created because it is in conflict with the existing graph.
+Either use xref::indexes-for-search-performance.adoc[] instead, or remove or correct the offending nodes and then re-apply the constraint.
```
Comment (JPryce-Aklundh) [inline modules/ROOT/pages/constraints/examples.adoc]:
```suggestion
Either use xref::indexes-for-search-performance.adoc[] instead, or remove/correct the offending nodes and then re-apply the constraint.
```

File: modules/ROOT/pages/constraints/examples.adoc [hunk 2]
```diff
@@ -325,7 +325,7 @@ RETURN book1, book2
 
 _This feature was introduced in Neo4j 5.7._
 
-A relationship property uniqueness constraint ensures that all relationships with a particular relationship type have a set of defined properties whose combined value is unique when existing.
+A relationship property uniqueness constraint ensures that certain relationships have a set of certain properties whose combined value is unique when all properties exists on the relationship.
```
Comment (JPryce-Aklundh) [inline modules/ROOT/pages/constraints/examples.adoc]:
```suggestion
A relationship property uniqueness constraint ensures that certain relationships have a set of specified properties whose combined value is unique when all properties exist on the relationship.
```

File: modules/ROOT/pages/constraints/examples.adoc [hunk 3]
```diff
@@ -12,7 +12,7 @@ Examples of how to manage constraints used for ensuring data integrity.
 [[constraints-examples-node-uniqueness]]
 == Node property uniqueness constraints
 
-A node property uniqueness constraint ensures that all nodes with a particular label have a set of defined properties whose combined value is unique when existing.
+A node property uniqueness constraint ensures that certain nodes have a set of certain properties whose combined value is unique when all properties exists on the node.
```
Comment (JPryce-Aklundh) [inline modules/ROOT/pages/constraints/examples.adoc]: Double "certain" reads awkwardly. 

```suggestion
A node property uniqueness constraint ensures that certain nodes have a set of specified properties whose combined value is unique when all properties exist on the node.
```

File: modules/ROOT/pages/constraints/examples.adoc [hunk 4]
```diff
@@ -626,7 +626,7 @@ RETURN knows1, knows2
 [[constraints-examples-node-property-existence]]
 == Node property existence constraints
 
-A node property existence constraint ensures that all nodes with a certain label have a certain property.
+A node property existence constraint ensures that certain nodes have a certain property.
```
Comment (JPryce-Aklundh) [inline modules/ROOT/pages/constraints/examples.adoc]:
```suggestion
A node property existence constraint ensures that certain nodes have a specified property.
```

File: modules/ROOT/pages/constraints/examples.adoc [hunk 5]
```diff
@@ -595,8 +595,8 @@ CREATE (:Book {title: 'Soul Eater'})-[:SEQUEL_OF {order: 2, seriesTitle: 'Chroni
 CREATE CONSTRAINT series_title FOR ()-[sequel:SEQUEL_OF]-() REQUIRE (sequel.seriesTitle) IS UNIQUE
 ----
 
-In this case, the constraint cannot be created because it is violated by existing data.
-Either use xref::indexes-for-search-performance.adoc[] instead, or remove the offending relationships and then re-apply the constraint.
+In this case, the constraint cannot be created because it is in conflict with the existing graph.
+Either use xref::indexes-for-search-performance.adoc[] instead, or remove or correct the offending relationships and then re-apply the constraint.
```
Comment (JPryce-Aklundh) [inline modules/ROOT/pages/constraints/examples.adoc]:
```suggestion
Either use xref::indexes-for-search-performance.adoc[] instead, or remove/correct the offending relationships and then re-apply the constraint.
```

File: modules/ROOT/pages/constraints/examples.adoc [hunk 6]
```diff
@@ -1615,7 +1615,7 @@ RETURN part
 [[constraints-examples-node-key]]
 == Node key constraints
 
-A node key constraint ensures that all nodes with a particular label have a set of defined properties whose combined value is unique and all properties in the set are present.
+A node key constraint ensures that certain nodes have a set of certain properties whose combined value is unique and all properties in the set are present.
```
Comment (JPryce-Aklundh) [inline modules/ROOT/pages/constraints/examples.adoc]: Do we need to explain "the set"
```suggestion
A node key constraint ensures that certain nodes have a set of specified properties whose combined value is unique and all properties in the set are present.
```

File: modules/ROOT/pages/constraints/examples.adoc [hunk 7]
```diff
@@ -1879,16 +1879,16 @@ Node(0) with label `Actor` must have the properties (`firstname`, `surname`)
...
 .Query
 [source, cypher, role=test-fail]
 ----
 CREATE CONSTRAINT actor_born FOR (actor:Actor) REQUIRE (actor.born) IS NODE KEY
 ----
 
-In this case the node key constraint can not be created because it is violated by existing data.
-Either use xref::indexes-for-search-performance.adoc[] instead, or remove the offending nodes and then re-apply the constraint.
+In this case, the node key constraint cannot be created because it is in conflict with the existing graph.
+Either use xref::indexes-for-search-performance.adoc[] instead, or remove or correct the offending nodes and then re-apply the constraint.
```
Comment (JPryce-Aklundh) [inline modules/ROOT/pages/constraints/examples.adoc]:
```suggestion
Either use xref::indexes-for-search-performance.adoc[] instead, or remove/correct the offending nodes and then re-apply the constraint.
```

**Review by JPryce-Aklundh — COMMENTED** [review]
Thanks a lot @Hunterness - I have mainly suggested to replace "certain" when used twice in the same sentence

**Review by JPryce-Aklundh — APPROVED** [review]

**Pull request merged**